conf/application.conf

```diff
@@ -248,6 +248,10 @@ feedback-frontend {
   url = "/feedback"
 }
 
+income-tax-submission {
+    host = "http://localhost:9302"
```

Comment:
this should be in following format:
income-tax-submission-frontend {
      host = localhost
      port = 9302
}